interface/src/app/mlmodel/mlModelComponents.js

```diff
@@ -30,4 +31,45 @@ angular.module('adage.mlmodel.components', [
   }]
 })
 
+.component('mlModelView', {
+  templateUrl: 'mlmodel/view.tpl.html',
+  controller: ['GlobalModelInfo', function(GlobalModelInfo) {
+    this.modelInfo = GlobalModelInfo;
+  }]
+})
+
+.component('mlModelValidator', {
```

Comment:
I see. This component is small. Unless other modules may also need a service to validate a ML model, I tend to keep it as is.